trio/_util.py

```diff
@@ -224,10 +240,10 @@ def decorator(func: CallT) -> CallT:
     return decorator
 
 
-def fixup_module_metadata(module_name, namespace):
-    seen_ids = set()
+def fixup_module_metadata(module_name: str, namespace: dict[str, t.Any]) -> None:
```

Comment:
```suggestion
def fixup_module_metadata(module_name: str, namespace: dict[str, object]) -> None:
```

You end up passing it as an `object` to `fix_one` anyways.

Reply:
The main reason I didn't change this one is to ensure we don't have problems with variance issues on the caller side. `vars(module)` probably gives `Any` anyway though, so that wouldn't matter. Swapping to `Mapping` would also work too.

Reply:
Oh yeah mapping's probably a nicer signature.